### PR TITLE
WIP fix make volumetric source space for sphere

### PR DESCRIPTION
**Pull request by ktavabi**
@larsoner couple questions with this one:

1. Wasn't sure about what you discussed with @mdclarke wrt to `pos` argument for `setup_volume_source_space` but I went with `pos=7.0` rather than a dictionary. If you know what `pos[‘rr’]` and `pos[‘nn’]` should look like then I can change that.

2. LMK how you want to handle trans for volumetric sourcespace. `make_forward_solution` seems to expect `type` mne.transforms.Transform class and not a dict. This seems to contradict [docstring](https://martinos.org/mne/stable/generated/mne.make_forward_solution.html)

### CODE REVIEW

**Comment by larsoner** [comment]
I don't think we need to write the trans file. Why add that?

`mne.transforms.Transform` is a subclass of `dict` so while technically it's correct, that doc should probably be updated

**Comment by ktavabi** [comment]
> I don't think we need to write the trans file. Why add that?
IDK. Posterity? I assume I can use `Transform` subclass then. 

> doc should probably be updated
I'll start PR

**Comment by larsoner** [comment]
> I assume I can use Transform subclass then.

Yes you can. But I wouldn't bother, in this case you can directly use `None`:

https://mne-tools.github.io/stable/generated/mne.make_forward_solution.html#mne.make_forward_solution

**Comment by ktavabi** [comment]
If `trans=None` then we shouldn't need this either, correct?

```
trans = {'from': FIFF.FIFFV_COORD_HEAD,
                     'to': FIFF.FIFFV_COORD_MRI,
                     'trans': np.eye(4)}
```

**Comment by larsoner** [comment]
right

File: .gitignore [hunk 1]
```diff
@@ -40,8 +40,8 @@ nosetests.xml
 .idea/
 
 # subject files
-examples/funloc/subj*/**/
-examples/funloc/subj_*/**/*.html
+/examples/funloc/subj01/
+/examples/funloc/subj02/
```
Comment (larsoner) [inline .gitignore]: these changes look incorrect -- why add the leading slash, and why change to `subj01` when the name is `subj_01`?

Comment (ktavabi) [reply .gitignore]: Sorry that's a typo from my end. The leading slash was inserted by pycharm.

**Comment by larsoner** [comment]
LGTM, good to go from your end @ktavabi ?

**Comment by ktavabi** [comment]
Worked with funloc data. @mdclarke latest commit work for you?

**Comment by mdclarke** [comment]
Yep! all good. Thanks!

**Comment by larsoner** [comment]
Thanks @ktavabi

**Pull request merged**